mmeds/server.py

```diff
@@ -855,6 +855,15 @@ def run_analysis(self, access_code, analysis_method, config, runOnNode=None):
             if runOnNode and not check_privileges(self.get_user(), self.testing):
                 raise err.PrivilegeError('Only users with elevated privileges may run analysis directly')
 
+            with Database(testing=self.testing, owner=self.get_user()) as dbase:
+                studies = dbase.get_all_user_studies(self.get_user())
```

Comment:
There are method in the Database class for getting the study directly by the access code

Reply:
I wasn't able to find these, do you know what they're named?